ui/selectors/permissions.js

```diff
@@ -264,22 +260,10 @@ export function getLastConnectedInfo(state) {
   }, {});
 }
 
-export function getPermissionsMetadataHostCounts(state) {
-  const metadata = getPermissionDomainsMetadata(state);
-  return Object.values(metadata).reduce((counts, { host }) => {
-    if (host) {
-      if (counts[host]) {
-        counts[host] += 1;
-      } else {
-        counts[host] = 1;
-      }
-    }
-    return counts;
-  }, {});
-}
-
```

Comment:
We're no longer storing the `host` in the subject metadata, and this selector was already quite pointless anyway.